src/main/java/uk/gov/hmcts/cmc/ccd/exception/MappingException.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.cmc.ccd.exception;
+
+public class MappingException extends RuntimeException {
```

Comment:
I don't think we need this here, I's used in mappers so need there only. please remove.